.eslintrc.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
we used jshint and jscs before which have been deprecated and fail miserably on linting es8 so I had to switch to eslint. This eslint config is for the server and files that aren't in the client.